commcare_export/writers.py

```diff
@@ -210,7 +210,7 @@ def write_table(self, table):
             assert self.tables[table.name].headings == list(table.headings)
 
         self.tables[table.name].rows.extend(
-            [[to_jvalue(v) for v in row] for row in table.rows]
+            to_jvalue(v) for row in table.rows for v in row
```

Comment:
I think this did the wrong thing (flattened the list instead of adding each row as its own list). going to revert unless you intended that to happen for some reason

Reply:
Oh, oops. I totally misunderstood what this was doing. Should be
```py
[to_jvalue(v) for v in row] for row in table.rows
```

Reply:
roger. you just wanted to remove the redundant braces, yeah?